runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -169,7 +169,7 @@ J9::Z::TreeEvaluator::inlineEncodeASCII(TR::Node *node, TR::CodeGenerator *cg)
    generateRILInstruction(cg, TR::InstOpCode::SLFI, node, numCharsLeftToProcess, 16);
 
    // Branch back up if we still have more than 16 characters to process.
-   generateS390CompareAndBranchInstruction(cg, TR::InstOpCode::C, node, numCharsLeftToProcess, 15, TR::InstOpCode::COND_BH, processMultiple16CharsStart, false, false);
+   generateS390CompareAndBranchInstruction(cg, TR::InstOpCode::C, node, numCharsLeftToProcess, 16, TR::InstOpCode::COND_BH, processMultiple16CharsStart, false, false);
```

Comment:
Is this intended? Or you changed in wrong place ?

Reply:
I think there's a misunderstanding here. Is your suggestion here that we unconditionally branch back to the beginning of this loop and then only do the check there for how many characters is left to process?

Reply:
Yeah I changed in the wrong place. Sorry!

Reply:
Fixed: d9181982c95ec80c9b3dfc6f058bb22688d4191b